README.md

```diff
@@ -1 +1,175 @@
-# Conference-page
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Conference page project] <a name="about-project"></a>
+
+> Conference page project 
+
+**[Conference page]** is a html and css project that contain a portfolio of me and with some colors and backgrounds
+
+## 🛠 Built With <a name="built-with">
+
+- HTML
+- CSS
+- Javascript
+
+</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">
+      HTML 
+     </a></li>
+    <li><a href="#">
+      CSS
+     </a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[add Contact me section]**
+- **[add add changes and styles]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- browser like google chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- clone the project using by running the following command :
+
+`git clone`
+`https://github.com/StevenWafeek/Conference-page`
+
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **steven wafeek**
+
+- GitHub: [StevenWafeek](https://github.com/StevenWafeek)
+- Twitter: [steven_hakeem](https://twitter.com/steven_hakeem)
+- LinkedIn: [steven wafeek
+](https://www.linkedin.com/in/steven-wafeek-b629341b6/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  **[Make a profissional capstone]**
+-  **[Make more details ]**
+-  **[work Efficiency]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/StevenWafeek/Conference-page/)
```

Comment:
Nice job on the README 📖 but consider the following to improve it:
- [ ] The live link 🔗  you have for now is not working, it's only a link going to the GitHub repo
- [ ] On that same note the project should be deployed using GitHub Pages 🚀 and the link should be added in that section of the README ☝🏽